cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,275 @@
+# Dynamic Objects (Only Supported in JS Objects)
+
+## [Supported Dynamic Glue Codes](../../support/step_definitions/dynamicCalls.md)
+
+- we test the '(.+)' getters and setters(?: '(._?)'(?: to '(._?)')?)?
+- 1st party app invokes the '(.+)' API to set( invalid)? value
+- '(.+)' invokes the '(.+)' get API
+- '(.+)' registers for the '(.+)' event
+- '(.+)' platform responds to '(.+)' (get|set) API(?: with '(.+)')?
+- '(.+)' platform (triggers|does not trigger) '(.\*?)' event(?: with '(.+)')?
+
+## Background
+
```

Comment:
Background can be 

FCS supports dynamic firebolt JS objects to be used across multiple examples in a scenario outline. This is applicable for test cases where we can validate setter response, getter response, event response etc using a single JS object. More details outline below

Reply:
done